### PR TITLE
do not require ssh to clone from source

### PR DESCRIPTION
**Pull request by anarcat**
Closes: #389

### CODE REVIEW

File: doc/install.txt [hunk 1]
```diff
@@ -135,7 +135,7 @@ Manual setup for OSX systems
    Now install py2app:
    $ sudo easy_install -U py2app
 
-5. git clone git@github.com:linkchecker/linkchecker.git linkchecker
+5. git clone https://github.com/linkchecker/linkchecker.git
```
Comment (mgedmin) [inline doc/install.txt:138]:
```suggestion
5. git clone https://github.com/linkchecker/linkchecker
```

I mean, either works, but the version without .git can also be opened in a web browser.

Well, both can be opened in a web browser, but the .git-less version is shorter and prettier.

**Comment by anarcat** [comment]
wtf is up with travis failing on apt install... gah... rerunning checks.

**Comment by mgedmin** [comment]
> wtf is up with travis failing on apt install... gah... rerunning checks.

I did that already, it did not help :/

**Comment by anarcat** [comment]
> > wtf is up with travis failing on apt install... gah... rerunning checks.
> 
> I did that already, it did not help :/

what the fuck. it's just a docs change... :(

**Pull request merged**